### ISSUE TITLE
Decode REG+REG*X as REG*(X+1)

### ISSUE THREAD

**Issue by AaronOpfer** [eteran/edb-debugger#678]
The instruction `MOVZX EAX, [RAX+RAX*4+0x5CFEA60]` appeared in an application I was debugging and, while I understand the instruction's bytecode translates literally to this instruction, I think it would make sense to instead represent this as `MOVZX EAX, [RAX*5+0x5CFEA60]`.

**Comment by eteran** [comment]
I agree, that this is a capstone "issue".  And I use the word issue lightly because capstone is technically doing the right thing and what you suggest is merely a visual simplification.
